packages/isomorphic/src/utils/browser.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  bytesToHex as nobleBytesToHex,
+  hexToBytes as nobleHexToBytes,
+  randomBytes as nobleRandomBytes,
+} from '@noble/hashes/utils'
+import { BytesToHexFn, HexToBytesFn, RandomBytesFn } from './types'
+
+// Typed to ensure uniformity between node and browser implementations and docs
+/* eslint-disable func-style, eslint-comments/disable-enable-pair -- see above */
```

Comment:
I don't think the `estlint-comments/disable-enable-pair` shouldn't be ignored. You should add the re-enable line below the `randomBytes` function so that any future additions to this file don't accidentally get included in the rule ignoring (or you should set the rule for the whole file if that's the intent)

Reply:
I did this to type all functions.  Any future functions should do the same.  In this case I wanted the explanation to be in the file as opposed to an eslint config.